lua/epics.lua

```diff
@@ -111,6 +111,25 @@ function M.setup(opts)
 
 	M.options = vim.tbl_extend("force", M.options, opts)
 
+	vim.filetype.add {
+		extension = {
+			ioc = "make",
+			Db = "make",
+		},
+		filename = {
+			["RULES.Db"] = "make",
+			["RULES.ioc"] = "make",
+			["CONFIG_STIE"] = "make",
+			["CONFIG_STIE.local"] = "make",
+			["CONFIG_STIE_ENV"] = "make",
+		},
```

Comment:
Since those paths are matched in `pattern`, is it possible to remove them?

Reply:
What I consider here is that user may copy these files to there own folders, so I think we should keep them.

Reply:
Sure!

Then:

```suggestion
		filename = {
			["RULES.Db"] = "make",
			["RULES.ioc"] = "make",
			["CONFIG_SITE"] = "make",
			["CONFIG_SITE.local"] = "make",
			["CONFIG_SITE_ENV"] = "make",
		},
```